Dockerfile.build

```diff
@@ -7,3 +7,6 @@ COPY dev-requirements.txt /tmp/dev-requirements.txt
 RUN pip install -r /tmp/dev-requirements.txt
 
 RUN rm /tmp/*requirements.txt
+
+RUN apt-get update
```

Comment:
why do u need to update? It's just a waste of time, no?

Reply:
Ill remove it